test/requisite/api_model_test.rb

```diff
@@ -11,139 +10,151 @@ class DummyApiModel < Requisite::ApiModel
       response = DummyApiModel.new
       def response.a; 'A'; end
       def response.b; 2; end
-      response.to_hash.must_equal( :a => 'A', :b => 2 )
-      response.must_respond_to :a
-      response.must_respond_to :b
+      _(response.to_hash).must_equal( :a => 'A', :b => 2 )
```

Comment:
These changes fix deprecation warnings

Example:
```
DEPRECATED: global use of must_equal from /home/sean/src/requisite/test/requisite/api_user_test.rb:130. Use _(obj).must_equal instead. This will fail in Minitest 6.
```